index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+          <h3>Welcome</h3>
+          <img class="menu-icon" src="images/Union.png" alt="menu" />
+        </nav>
+      </header>
+
+      <!--First content-->
+    <section>
+        <div class="content">
+          <h1>
+            Hey there. I’m Donard <br /><span>I’m a software developer</span>
+          </h1>
+          <p>
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <ul class="content-list">
+          <li>
+            <a href="https://twitter.com/donardgolaj"
+              ><img src="images/twitter.png" alt="twitter"
+            /></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/donard-golaj/"
+              ><img src="images/Linkedin.png" alt="linkedin"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/medium.png" alt="medium" /></a>
+          </li>
+          <li>
+            <a href="https://github.com/Donard97"
+              ><img src="images/github.png" alt="github"
+            /></a>
+          </li>
+          <li>
+            <a href="#"><img src="images/angellist.png" alt="angellist" /></a>
+          </li>
+        </ul>
+      </section>
+      <button id="btn" type="button">
+        <img src="images/Disabled.png" alt="button" />
+      </button>
+  
+    
```

Comment:
- You left an extra empty line in your code that is serving no purpose. To ensure a cleaner code let's remove the extra empty line.